src/redux/greetingSlice.js

```diff
@@ -0,0 +1,39 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+export const getGreeting = createAsyncThunk('get/message',
+  async () => {
+    try {
+      const response = await fetch('http://localhost:3000/api/greeting');
+      console.log(response);
+      const result = await response.json();
+      return result;
+    } catch (error) {
+      console.error(error);
```

Comment:
- You have done brilliantly so far, but please note that it is bad practice to leave `consle.log` statements in your code, this makes your code look unprofessional, I suggest you remove all `console.log` statements from your code. Please fix this 🙏